mountpoint-s3-fs/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Allow changing log level dynamically with `USR2` signal. See [Changing logging verbosity at runtime](https://github.com/awslabs/mountpoint-s3/blob/main/doc/LOGGING.md#changing-logging-verbosity-at-runtime) for more details. ([#1367](https://github.com/awslabs/mountpoint-s3/pull/1367))
 * Use the Runtime type in the prefetcher. ([#1382](https://github.com/awslabs/mountpoint-s3/pull/1382))
 * Move CliArgs and related functions to the mountpoint-s3 crate. ([#1401](https://github.com/awslabs/mountpoint-s3/pull/1401))
+* Update underlying S3 client version.
```

Comment:
not included link here since it's not clear which PR is most important. we could include the PR that actually bumps the patch version in Cargo (but that's this PR!)

Reply:
I think its important to include this in the changelog here, otherwise the changelog would have no content if we only release client changes and full MP release.

Reply:
Agree. Not against linking to this PR either.

Reply:
Will link anyway for consistency since I'm already updating PR